ci/cloudbuild/dockerfiles/fedora-36-cmake.Dockerfile

```diff
@@ -61,10 +61,13 @@ RUN curl -sSL https://pkgconfig.freedesktop.org/releases/pkg-config-0.29.2.tar.g
     ldconfig
 ENV PKG_CONFIG_PATH=/usr/local/lib64/pkgconfig:/usr/local/lib/pkgconfig:/usr/lib64/pkgconfig
 
+# We expose `absl::optional<>` in our public API. An Abseil LTS update will
+# break our `check-api` build, unless we disable the inline namespace.
 WORKDIR /var/tmp/build
 RUN curl -sSL https://github.com/abseil/abseil-cpp/archive/20220623.1.tar.gz | \
     tar -xzf - --strip-components=1 && \
     sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h" && \
```

Comment:
Unrelated, starting with this release of Abseil we should not need this.  Abseil install will do that themselves:

https://github.com/abseil/abseil-cpp/blob/75d2525117c8da93840ab256f07b191086fd7cbb/CMakeLists.txt#L229-L243